.ci.yaml

```diff
@@ -1091,9 +1091,10 @@ targets:
   # TODO(stuartmorgan): Remove "- packages" from all task names once
   # flutter/plugins is merged into this repo and turned down; it's here only
   # because names must be unique across all flutter repositories.
-  - name: Mac_arm64 macos_platform_tests master - packages
+  - name: Mac_x64 macos_platform_tests master - packages
```

Comment:
For science, I wonder if these are hanging because they are on arm bots...

Reply:
We shouldn't change this without changing the build-all platform the other direction. And we should do it with new bots so we don't lose all macOS coverage for 1-2 days while this is in bringup.

Reply:
Reverted this change since the hang is fixed upstream and wasn't related to the arch anyway.